components/Drawer/src/index.svelte

```diff
@@ -8,34 +9,64 @@
 	export let value = false;
 	export let target: null | HTMLElement = null;
 	export let cls = '';
-	export let attrs = {}
+	export let attrs = {};
+	export let footer = false;
+
 	const dispatch = createEventDispatcher();
+	$: showInner = value;
+
+	const close = () => {
+		showInner = !showInner;
+		dispatch('close', showInner);
+	};
+
 	const toggleClose = () => {
-		dispatch('close');
+		close();
+	};
+
+	const cancel = () => {
+		dispatch('cancel');
```

Comment:
You added two events, you seem to want them to be exposed api, if so, you need to explain in the documentation